src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,51 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_VALUE = 10;
+    private static final int VALUE_NOT_FOUND = -1;
+    private K[] keyList;
+    private V[] valueList;
```

Comment:
```suggestion
    private K[] keys;
    private V[] values;
```
moreover it's not even a list, but array